stage1_fly/aci/aci-manifest.in

```diff
@@ -0,0 +1,33 @@
+{
+    "acKind": "ImageManifest",
+    "acVersion": "0.7.3",
+    "name": "coreos.com/rkt/stage1-fly",
+    "labels": [
+        {
+            "name": "version",
+            "value": "@RKT_STAGE1_VERSION@"
+        },
+        {
+            "name": "arch",
+            "value": "amd64"
+        },
+        {
+            "name": "os",
+            "value": "linux"
+        }
+    ],
+    "annotations": [
+        {
+            "name": "coreos.com/rkt/stage1/run",
+            "value": "/run"
+        },
+        {
+            "name": "coreos.com/rkt/stage1/enter",
+            "value": "/enter"
```

Comment:
I think that we should allow to omit the dictionaries for enter and gc instead of allowing empty values.